### ISSUE TITLE
di:compile error after composer require dotdigital/dotdigital-magento2-os-package (M2.3.5 / PHP7.2)

### ISSUE THREAD

**Issue by ealforque** [dotmailer/dotmailer-magento2-extension#597]
Hi ,

Magento 2.3.5(PHP 7.2)

Executed `composer require dotdigital/dotdigital-magento2-os-package` and `php bin/magento di:compile`

`Parse error:  syntax error, unexpected ')' in /var/www/magento2-dev/codebase/vendor/dotdigital/dotdigital-magento2-extension/Model/Sync/Importer/ImporterReportHandler.php on line 189`

Temporarily created and applied a patch to fix.

Regards,
Eric

**Comment by sta1r** [comment]
@ealforque what version of the metapackage do you have? Also what version of this module? In latest 4.19.4 L189 has no ')': https://github.com/dotmailer/dotmailer-magento2-extension/blob/master/Model/Sync/Importer/ImporterReportHandler.php#L189

Try `composer show dotdigital/*`.

**Comment by ealforque** [comment]
Hi @sta1r, here is the what is installed after doing `composer require dotdigital-magento2-os-package` 
```
dotdigital/dotdigital-magento2-extension               4.14.0 Dotdigital for Magento 2
dotdigital/dotdigital-magento2-extension-chat          1.6.0  Dotdigital for Magento 2: Chat module
dotdigital/dotdigital-magento2-extension-chat-graph-ql 1.1.0  Dotdigital for Magento 2: Chat GraphQl module
dotdigital/dotdigital-magento2-extension-graph-ql      1.1.0  Dotdigital for Magento 2: Core GraphQl module
dotdigital/dotdigital-magento2-extension-sms           1.3.1  Dotdigital for Magento 2: SMS module
dotdigital/dotdigital-magento2-os-package              1.0.0  Dotdigital package for Magento Open Source
```

**Comment by sta1r** [comment]
Yes, these are old. You have to get the newest metapackage from the marketplace: https://marketplace.magento.com/dotdigital-dotdigital-magento2-os-package.html. If your auth keys match the account that was used to make the purchase, you will be able to access the latest code.

**Comment by ealforque** [comment]
@sta1r We tried doing composer require dotdigital/dotdigital-magento2-os-package:1.5.4 and it installed the latest files. Closing this issue now. Thank you.
